CMakeLists.txt

```diff
@@ -24,6 +24,12 @@ cmake_dependent_option(LIBUV_BUILD_BENCH
   "Build the benchmarks when building unit tests and we are the root project" ON
   "LIBUV_BUILD_TESTS" OFF)
 
+# Qemu Build
+option(QEMU "build for qemu" OFF)
+if(QEMU)
+  add_definitions(-D__QEMU=1)
```

Comment:
The tests are checking for `__QEMU__`.

Reply:
`return` in test looks like not working, do I miss something ?

Reply:
You'll want to use something similar to:

`RETURN_SKIP("Test does not currently work in QEMU");`

Just search for `RETURN_SKIP` in the tests.

Reply:
Tests are supposed to return a value, one of: https://github.com/libuv/libuv/blob/72fe3543feb23ae555e08628b70a3fae4da5706c/test/task.h#L253-L256

I suggest using the `RETURN_SKIP` macro: https://github.com/libuv/libuv/blob/72fe3543feb23ae555e08628b70a3fae4da5706c/test/task.h#L263-L268